### PR TITLE
Add color selection boxes to new game screen

### PR DESCRIPTION
**Pull request by sekelsta**

### CODE REVIEW

File: src/render/GameMenuScreen.java [hunk 1]
```diff
@@ -31,7 +32,7 @@ public void blit(double screenWidth, double screenHeight)
         for (GuiElement item : selectable) {
             int xPos = ((int)screenWidth - item.getWidth()) / 2;
```
Comment (earlyraven) [inline src/render/GameMenuScreen.java:33]: Should this be:
            int xPos = (int)((screenWidth - item.getWidth() / 2);
instead?

**Review by earlyraven — APPROVED** [review]

**Comment by earlyraven** [comment]
Looks good.  See comment regarding int casting.

File: src/render/GameMenuScreen.java (hunk 1 above)
Comment (sekelsta) [reply src/render/GameMenuScreen.java:33]: No it should not, the rounding is supposed to happen before the division by two

Comment (sekelsta) [reply src/render/GameMenuScreen.java:33]: On further inspection, this change is actually fine. I was thinking of the two-column logic from the other screen.

**Pull request merged**